api/client/samples/anomaly_detection/sample_anomaly_detection.ipynb

```diff
@@ -0,0 +1,799 @@
+{
```

Comment:
<blockquote>Now we need a function to help us <strong>obtain</strong> the best data series for multiple locations&nbsp;We want to choose the <strong>series</strong> with the best rank so we use this handy method to retrieve the best</blockquote><p>For showcase purposes, it may be best to just hardcode in the entity ids? We can say that we found the ids through the web app and provide additional detail in that process if necessary. The only remaining two ids we would need for this call is the source_id and the frequency_id. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/146/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>I kind of like sharing a method for them to search for ideal series though! what do you think?</p>

Reply:
I like it too. Sometimes I feel like get_data_series is a function that is more for the advanced user for our API, as it takes some time to understand what it does and how it differs from get_data_points. But totally fine to leave it as an internal implementation of the notebook. Maybe just add a comment or two sharing why it's used (in case no comment exists already).